src/routes/exercise/controllers.ts

```diff
@@ -3,6 +3,7 @@ import { Request, Response } from 'express';
 import { prisma } from 'src/config/prisma';
 import { CustomError } from 'src/types/custom-error';
 import { getActionSuccessMsg, missingId, notFound } from 'src/utils/messages';
+import { exerciseSchema } from 'src/validations/exerciseValidations';
```

Comment:
I would prefer that the file should be called exercise since it is already in a directory called validations.